### PR TITLE
chore: update unit tests in new format for destinations (af, algolia, appcues, blueshift, branch) 

### PR DESCRIPTION
**Pull request by ujjwal-ab**
## Description of the change

update unit tests in new format for destinations (af, algolia, appcues, blueshift, branch) 

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: test/integrations/destinations/af/router/data.js [hunk 1]
```diff
@@ -0,0 +1,398 @@
+const data = [
+    {
+      "name": "af",
+      "description": "Test 0",
```
Comment (lvrach) [inline test/integrations/destinations/af/router/data.js:4]: I would suggest using a better description for the test. Without it, people will have a hard time maintaining them, especially given the verbose input.

File: test/integrations/destinations/af/router/data.js [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+                  "Enabled": true
+                }
+              }],"destType": "af"},
+            "method": "POST"
+          },
+          "pathSuffix": ""
+        },
+        "output": {
+          "response": {
+            "status": 200,
```
Comment (lvrach) [inline test/integrations/destinations/af/router/data.js:342]: Are there any negative test cases for this destination?
For example, an input that should return 4xx.

File: test/integrations/destinations/appcues/router/data.js [hunk 3]
```diff
@@ -0,0 +1,238 @@
+const data = [
+    {
+      "name": "appcues",
+      "description": "Test 0",
```
Comment (krishna2020) [inline test/integrations/destinations/appcues/router/data.js:4]: Can we spend some time & update correct descriptions ?

Comment (ujjwal-ab) [reply test/integrations/destinations/appcues/router/data.js:4]: @krishna2020 I had looked into this. But it is pretty difficult to know what this test case is for through them, as most of them are pretty similar.

**Comment by devops-github-rudderstack** [comment]
This PR is considered to be stale. It has been open for 20 days with no further activity thus it is going to be closed in 7 days. To avoid such a case please consider removing the stale label manually or add a comment to the PR.

**Pull request closed without merging**